analyze.sh

```diff
@@ -14,11 +14,12 @@ if [ -d "$ARTIFACT_DIR" ]; then
   cp public/dist/report.html "${ARTIFACT_DIR}"
 fi
 
+MAX_BYTES=2621440 # ~2.5 MiB
```

Comment:
You can't do floating point calculations in bash (2.5 * 1024 * 1024), so I just hard-coded the value

Reply:
@spadgett i ran into this too...was wondering how you wanted this to be modified...